doc/manual.rst

```diff
@@ -3479,11 +3479,37 @@ Accessing trading fee rates should be done via the ``.markets`` property, like s
 
 .. code:: javascript
 
-   exchange.markets['ETH/BTC']['taker'] // taker fee rate for ETH/BTC
    exchange.markets['BTC/USD']['maker'] // maker fee rate for BTC/USD
 
+The ``.markets`` property may contain additional fee related information:
```

Comment:
Can you plz move these edits to wiki/Manual.md ? The .rst file is transpiled from .md: 

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#transpiled-generated-files

Thanks!